src/poolman.h

```diff
@@ -0,0 +1,9 @@
+#ifndef __BITCOIN_POOLMAN_H__
```

Comment:
Also misses license and correct header define ``#ifndef BITCOIN_POOLMAN_H`` (and at the file end).